aea/manager.py

```diff
@@ -577,6 +577,49 @@ def get_agent_alias(self, agent_name: str) -> AgentAlias:
             raise ValueError(f"Agent with name {agent_name} does not exist!")
         return self._agents[agent_name]
 
+    def get_overridable_config(self, agent_name: str) -> Dict[str, Any]:
+        """
+        Get overridable config of the agent.
+
+        :param agent_name: str agent name.
+
+        :return: dict overridable config.
+        """
+        alias = self.get_agent_alias(agent_name)
+        config = alias.builder.loader.load_agent_config_from_json(alias.config)
+        agent_config = {
+            key: value
+            for key, value in config.json.items()
+            if key in config.FIELDS_ALLOWED_TO_UPDATE
+        }
+        # TODO: Get agent components overridable config the same way.
+        # we miss AgentConfig class functionality to do that.
```

Comment:
All the values in a component (excluding public id and type) are by definition overridable.

Reply:
We need to have a property that lists all the fields except of that two as overridable.